client/components/deposits-overview/style.scss

```diff
@@ -23,34 +23,40 @@
 		}
 	}
 	&__notices {
-		padding: 0 24px;
 		.wcpay-banner-notice.components-notice {
 			margin: 0;
 		}
 	}
+
+	// Override extraneous CardBody vertical padding -
+	// we use multiple stacked CardBody components
+	// so table CardDivider is full widh.
+	.components-card__body.wcpay-deposits-overview__table__container {
+		padding-bottom: 0;
+		padding-top: 0;
+
+		.wcpay-banner-notice.components-notice {
+			margin: 0;
+			&:last-child {
+				margin-bottom: 16px;
```

Comment:
I might have given you some slightly bad advice regarding this. 

This margin bottom is being applied to the notices up in the next deposit section, when it was only intended to be applied to the notice in the deposit history table if its the last child. eg:

<img width="702" alt="Screenshot 2023-04-12 at 3 23 44 pm" src="https://user-images.githubusercontent.com/8490476/231358574-0bf08cdc-5293-4695-b33c-b15c79661603.png">

Notice the gap after the capital loan notice and the Deposit section heading.

It's supposed to look like this: 

<img width="700" alt="Screenshot 2023-04-12 at 3 25 07 pm" src="https://user-images.githubusercontent.com/8490476/231358837-c94a730c-52ef-4530-8c59-a508551e5dfd.png">

I don't have a good idea on how to resolve that. Either change this css selector to be more targeted so it only applies to the notices in the deposit history table. 

This is the scenario this style is meant to apply to. 
<img width="704" alt="Screenshot 2023-04-12 at 3 29 39 pm" src="https://user-images.githubusercontent.com/8490476/231359566-7ed9e1f3-51d3-4870-909d-71f627d4c146.png">

Reply:
Looking over this style sheet there's a lot of instances (at least 3) of `margin: 0` being applied to the `.wcpay-banner-notice.components-notice` element. We might need to do a broader review of those notice styles tbh so let me know if this should be taken out of this issue.

Reply:
Thanks, I'll take a look at this today. I didn't test all the notices so thanks for catching :)

Reply:
I've hacked in all notices, and I _think_ it's looking good - though maybe there's a combination that's broken. I still need to test the "1 deposit which is in transit" case. 

![Screenshot 2023-04-13 at 10 28 10 AM](https://user-images.githubusercontent.com/4167300/231600260-8c2cda7f-392d-4fa9-bb74-0634f4ab6700.png)

Reply:
The extra padding applies only when the Capital loan notice is shown. When there are other notices it looks right. 

<img width="692" alt="Screenshot 2023-04-13 at 9 50 35 am" src="https://user-images.githubusercontent.com/8490476/231610470-2f09a6b0-5322-4550-923f-4af99c28eca2.png">

The amount of padding between the notice and the **Deposit history** heading is an extra 16px. 

This is what it looks like on `develop`: 
<img width="702" alt="Screenshot 2023-04-13 at 9 52 58 am" src="https://user-images.githubusercontent.com/8490476/231610690-60459e6c-7d82-40b2-9a5d-d569758cbdc3.png">

The is happening because the capital loan notice is embedded in the `wcpay-deposits-overview__table__container` and so gets applied [this last child styling](https://github.com/automattic/woocommerce-payments/blob/fix/use-cardbody-component-for-standard-padding/client/components/deposits-overview/style.scss#L40-L42) whereas the deposit delay notice is in it's own div.

I think @brucealdridge has been looking at padding between these notices as part of https://github.com/Automattic/woocommerce-payments/issues/6002 so it might be handled as part of that.

Reply:
> The is happening because the capital loan notice is embedded in the wcpay-deposits-overview__table__container and so gets applied [this last child styling](https://github.com/automattic/woocommerce-payments/blob/fix/use-cardbody-component-for-standard-padding/client/components/deposits-overview/style.scss#L40-L42) whereas the deposit delay notice is in it's own div.

Maybe we can consistently put these two notices in same place in DOM – sounds like that would fix it.